src/main/java/org/jabref/gui/JabRefDialogService.java

```diff
@@ -246,6 +235,19 @@ public void showErrorDialogAndWait(FetcherException fetcherException) {
         }
     }
 
+    private String getContentByCode(int statusCode) {
+        return switch (statusCode) {
+            case 401 ->
+                "Access denied. You are not authorized to access this resource. Please check your credentials and try again. If you believe you should have access, please contact the administrator for assistance.";
```

Comment:
Please wrap in `Localization.lang`. See https://devdocs.jabref.org/code-howtos/localization.html#localization-in-java-code for reasons.